status-im/status-react
# fix: banner images/colors in shell

fixes #16112 

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
Incorrect banner images/colors on the shell screen

### Review notes
You can check it on the shell screen.
For now, 1:1 chat card is not available to show exact colors since it needs to be done in the back-end side.

### Testing notes

![simulator_screenshot_61C72BF0-E0A5-4294-AC06-AA8094EE91D1](https://github.com/status-im/status-mobile/assets/39961806/d34744c4-655c-4766-b135-9d7ed7b15b76)

#### Platforms


- Android
- iOS
- macOS
- Linux
- Windows

#### Areas that maybe impacted
Shell screen

### Steps to test


- Open Status
- Click the `Jump To` button in the bottom center.
- You can check several shell cards with proper images/colors except for 1:1 chat.



status: ready
